src/api.js

```diff
@@ -13,3 +13,40 @@ export const getTopics = () => {
     return response.data.topics;
   });
 };
+
+export const getArticleById = (articleId) => {
+  return api
+    .get(`/articles/${articleId}`)
+    .then((response) => {
+      return response.data.article;
+    })
+    .catch((error) => {
+      throw error;
+    });
```

Comment:
You need to catch at the end of your chain. Do you need to catch here, only to throw straight away?